workspaces/web/src/components/NavBar.svelte

```diff
@@ -26,7 +26,14 @@
   <div class="navbar-end">
     <div class="navbar-item">
       <div class="buttons">
-        <GitHubButton />
+		<Button href="http://github.com/sponsors/kantord" size="small" outlined inverted info>
+		<svg viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true" style="fill: #ea4aaa!important;margin-right: 5px">
```

Comment:
Please don't inline this svg. Instead, use an icon from font awesome.

We already have an icon component for this: https://github.com/kantord/LibreLingo/blob/master/workspaces/lluis/Icon.svelte

Here's an example of the icon component being used:
https://github.com/kantord/LibreLingo/blob/ccfaaccf958055069659e19654ad10bdc2002f6e/workspaces/web/src/components/GitHubButton.svelte#L50

If the icon component doesn't currently support the specific icon you need, you can add it here:
https://github.com/kantord/LibreLingo/blob/master/workspaces/web/src/client.js

You can browse the set of the available icons here: https://fontawesome.com/icons/search

If you can't find the same icon that you put here, you can choose whatever else that fits the purpose in your judgement!